lib/SQL/Translator/Parser/PostgreSQL.pm

```diff
@@ -842,8 +1052,8 @@ alter : alter_table table_id DROP /constraint/i NAME restrict_or_cascade ';'
 alter : alter_table table_id /owner/i /to/i NAME ';'
     { 1 }
 
-alter : alter_sequence NAME /owned/i /by/i column_name ';'
-    { 1 }
+# alter : alter_sequence NAME /owned/i /by/i column_name ';'
+#     { 1 }
```

Comment:
is this only temporarily removed? if so, can you add a TODO for that?

Reply:
I added TODO.